sklearn/gaussian_process/kernels.py

```diff
@@ -910,6 +940,7 @@ class Exponentiation(Kernel):
     >>> gpr.predict(X[:1,:], return_std=True)
     (array([635.5...]), array([0.559...]))
     """
+
```

Comment:
Could you revert all these new lines here and below?

Reply:
I think I am unclear by when you say revert all the new lines. Are you saying remove the extra space?

Reply:
Sorry, after further review I understand the issue and will revert the new lines.